### ISSUE TITLE
Show date of entry in each row

### ISSUE THREAD

**Issue by willhains** [willhains/Dictionary#14]

**Comment by GaborTheFisher** [comment]
I can take care of that.
Would like to propose to use `CoreData` for persisten insted of `NSUserDefaults`.

**Comment by willhains** [comment]
@GaborTheFisher Yes the current use of `NSUserDefaults` is a bit of an embarrassment. I did that as a quick temp solution and never got around to fixing it. Opened #34 for that — #29 and #14 (this issue) would I guess depend on it.

**Comment by GaborTheFisher** [comment]
@willhains, just got back from holiday, so will have time look into this. Will most likely be using `CoreData` for this.

**Comment by willhains** [comment]
Sounds good. Hope you had a nice holiday! :)

**Comment by GaborTheFisher** [comment]
Just a quick update on this and #34.
Have added the `CoreData` stack to the project and a way of migrating entried from `NSUserDefaults`.
I'm able to display existing entries using `CoreData` and a subclass of `NSManagedObject`.
Should be able to raise PR in the next couple of days.
